internal/pkg/utils/strings.go

```diff
@@ -0,0 +1,15 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import "strconv"
+
+// QuoteStringSlice adds quotation marks around each string in the string slice.
+func QuoteStringSlice(in []string) []string {
```

Comment:
I think this function and func[ QuoteSliceFunc ](https://github.com/aws/copilot-cli/blob/8ab524dc945cd80f1889d44b7860ee44cc5170c3/internal/pkg/template/template_functions.go#L96) doing the same functionality. Do you think we can use the existing function? 

Also I think this is the right place of this string manipulation function but just that code is getting duplicated.

Reply:
Adopted option 1 here: https://github.com/aws/copilot-cli/pull/3751#discussion_r920389517!